intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -656,6 +656,66 @@ \subsection{Check your quota}
 slow down file operations on the cluster for all other users.
 \fi
 
+\section{Groups}
+\label{sec:vcs-groups}
+
+Groups are a way to manage who can access what data. A user can belong to multiple
+groups at a time. \strong{Groups can be created and managed without any interaction from
+the system administrators}.
+
+To change the group of a directory and it's underlying directories and files, you can use:
+
+\begin{prompt}
+%\shellcmd{chgrp -R groupname directory}%
+\end{prompt}
+
+\subsection{Joining an existing group}
+
+\begin{enumerate}
+    \item Get the group name you want to belong to.
+    \item Go to \url{https://account.vscentrum.be/django/group/new} and fill in the
+        section named ``Join group''. You will be asked to fill in the group name and a message for
+        the moderator of the group, where you identify yourself. This should look something
+        like \autoref{fig:group-join}.
+    \item After clicking the submit button, a message will be sent to the moderator
+    of the group, who will either approve or deny the request.
```

Comment:
Mention that you will be a member of the group shortly after the group moderator approves your request.